t/apicast-policy-chains-crash.t

```diff
@@ -0,0 +1,34 @@
+use lib 't';
+use Test::APIcast::Blackbox 'no_plan';
+
+$ENV{APICAST_ACCESS_LOG_FILE} = "$Test::Nginx::Util::ErrLogFile";
+$ENV{APICAST_POLICY_LOAD_PATH} = 't/fixtures/policies';
+
+repeat_each(1);
+run_tests();
+
+__DATA__
+
+=== TEST 1: policy chain with a policy that crashes on new()
+Policies that crash when initialized should be removed from the chain
+--- configuration
+{
+    "services": [
+      {
+        "id": 42,
+        "proxy": {
+            "policy_chain" : [
+              { "name" : "error_policy", "version" : "1.0.0" },
+              { "name" : "apicast.policy.echo" } 
+            ]
+        }
+      }
+    ]
+}
+--- request
+GET /test
+--- response_body
+GET /test HTTP/1.1
+--- error_code: 200
```

Comment:
the policy chain is corrupted and still returns 200 OK ? weird, isn't it?

Reply:
the request returns 200 because the expected behaviour is for the failing policy to be just removed from the policy chain silently, leaving the rest of the policy chain in working conditions. See [here](https://github.com/3scale/APIcast/pull/1350/files#diff-083e627e453a3a925ace595f76e2aee23b78d31bfae85eac4e71692f7716158aL247-L280) the original test (just moved in this new file as explained above)

Reply:
thanks for the explanation